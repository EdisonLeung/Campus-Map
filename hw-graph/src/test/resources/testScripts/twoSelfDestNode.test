# Creates 2 nodes and tests adding 2 edges that reference the node the came from

# Create graph
CreateGraph twoSelfDest

# Add 2 nodes
AddNode twoSelfDest a
AddNode twoSelfDest b

# Add 2 edges that point to the node they came from
AddEdge twoSelfDest a a AA
AddEdge twoSelfDest b b BB

# Print nodes and children
ListNodes twoSelfDest
ListChildren twoSelfDest a
ListChildren twoSelfDest b