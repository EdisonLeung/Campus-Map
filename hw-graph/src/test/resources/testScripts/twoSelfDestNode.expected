# Creates 2 nodes and tests adding 2 edges that reference the node the came from

# Create graph
created graph twoSelfDest

# Add 2 nodes
added node a to twoSelfDest
added node b to twoSelfDest

# Add 2 edges that point to the node they came from
added edge AA from a to a in twoSelfDest
added edge BB from b to b in twoSelfDest

# Print nodes and children
twoSelfDest contains: a b
the children of a in twoSelfDest are: a(AA)
the children of b in twoSelfDest are: b(BB)